config/default.yml

```diff
@@ -110,6 +110,12 @@ RSpec:
       - subject
       - subject!
 
+Metrics/BlockLength:
+  Enabled: true
```

Comment:
I don't think we need to explicitly enable it. Even though, probably it's disabled in user project's `.rubocop.yml` and this would override this, I can think of cases when common rules are extracted to a gem and it's `require:`'d in `.rubocop.yml` before or after `rubocop-rspec`, and this may inadvertently enable the disabled cop.